drawBot/context/imageContext.py

```diff
@@ -32,19 +32,25 @@ def _tiffCompressionConverter(value):
-    ("imagePNGInterlaced",           AppKit.NSImageInterlaced,              None,                      "Boolean value that indicates whether the image should be interlaced."),  # XXX doesn't seem to work
-    ("imageJPEGProgressive",         AppKit.NSImageProgressive,             None,                      "Boolean that indicates whether the image should use progressive encoding."),
-    ("imageGIFRGBColorTable",        AppKit.NSImageRGBColorTable,           _nsDataConverter,          "A bytes or NSData object containing the RGB color table."),
-]
+_nsImageOptions = {
+    # DrawBot Key                   NSImage property key                    converter or None          doc
+    "imageColorSyncProfileData":    (AppKit.NSImageColorSyncProfileData,    _nsDataConverter,          "A bytes or NSData object containing the ColorSync profile data."),
+    "imageTIFFCompressionMethod":   (AppKit.NSImageCompressionMethod,       _tiffCompressionConverter, "None, or 'lzw' or 'packbits', or an NSTIFFCompression constant"),
+    "imagePNGGamma":                (AppKit.NSImageGamma,                   None,                      "The gamma value for the image. It is a floating-point number between 0.0 and 1.0, with 0.0 being black and 1.0 being the maximum color."),
+    "imagePNGInterlaced":           (AppKit.NSImageInterlaced,              None,                      "Boolean value that indicates whether the image should be interlaced."),  # XXX doesn't seem to work
+    "imageJPEGCompressionFactor":   (AppKit.NSImageCompressionFactor,       None,                      "A float between 0.0 and 1.0, with 1.0 resulting in no compression and 0.0 resulting in the maximum compression possible"),  # number
+    "imageJPEGProgressive":         (AppKit.NSImageProgressive,             None,                      "Boolean that indicates whether the image should use progressive encoding."),
+    # "imageJPEGEXIFData":          (AppKit.NSImageEXIFData,                None,                      ""),  # dict  XXX Doesn't seem to work
+    "imageFallbackBackgroundColor": (AppKit.NSImageFallbackBackgroundColor, _nsColorConverter,         "The background color to use when writing to an image format (such as JPEG) that doesn't support alpha. The color's alpha value is ignored. The default background color, when this property is not specified, is white. The value of the property should be an NSColor object or a DrawBot RGB color tuple."),
+    "imageGIFDitherTransparency":   (AppKit.NSImageDitherTransparency,      None,                      "Boolean that indicates whether the image is dithered"),
+    "imageGIFRGBColorTable":        (AppKit.NSImageRGBColorTable,           _nsDataConverter,          "A bytes or NSData object containing the RGB color table."),
+}
+
+
+def getSaveImageOptions(options):
+    return ImageContext.saveImageOptions + [
+        (dbKey, _nsImageOptions[dbKey][-1]) for dbKey in options if dbKey in _nsImageOptions
+    ]
```

Comment:
`options` is unused, do you mean `options` instead of `ImageContext.saveImageOptions`?